### PR TITLE
Configurable image_repository

### PR DESCRIPTION
**Pull request by jakolehm**
By default pulls all the "core" images from `quay.io/kontena/`.

Todo:

- [x] kubernetes control plane
- [x] kubedns
- [x] etcd
- [x] heapster
- [x] pause
- [x] pharos-kubelet-proxy
- [x] weave
- [x] calico

### CODE REVIEW

**Comment by jakolehm** [comment]
@SpComb @jnummelin PTAL

File: lib/pharos_cluster.rb [hunk 1]
```diff
@@ -13,5 +13,5 @@ module Pharos
   KUBEADM_VERSION = ENV.fetch('KUBEADM_VERSION') { KUBE_VERSION }
   ETCD_VERSION = ENV.fetch('ETCD_VERSION') { '3.1.12' }
   DOCKER_VERSION = '1.13.1'
-  KUBELET_PROXY_VERSION = '0.3.5'
+  KUBELET_PROXY_VERSION = '0.3.6'
```
Comment (SpComb) [inline lib/pharos_cluster.rb]: Merge from #383?

Comment (jakolehm) [reply lib/pharos_cluster.rb]: Yes, it did not make sense to keep these separate (that's why I closed #383)

Comment (SpComb) [reply lib/pharos_cluster.rb]: Ah, version bump is for the image push to `quay.io`?

Comment (jakolehm) [reply lib/pharos_cluster.rb]: Yes.

**Comment by jnummelin** [comment]
Should we handle all the built-in addons from Quay.io too?

**Comment by jakolehm** [comment]
> Should we handle all the built-in addons from Quay.io too? 

Yes, but it should be a separate PR (needs more image plumbing)

**Pull request merged**